src/app/core/core.module.ts

```diff
@@ -18,11 +18,11 @@ import {
 
 export const MQTT_SERVICE_OPTIONS: IMqttServiceOptions = {
   hostname: `${environment.mqttUrl}`,
-  path: `${environment.mqttPathPrefix}${environment.mqttPath}`,
-  protocol: 'wss',
+  // path: `${environment.mqttPathPrefix}${environment.mqttPath}`,
+  protocol: 'ws',
   port: environment.mqttPort,
-  username: 'ui_client',
-  password: 'ud84jc23'
+  username: 'student',
+  password: 'passme'
 };
 
 @NgModule({
```

Comment:
you should maybe extract the user credentials into secrets.

Reply:
extracting the user credentials is planned for a future PR

Reply:
Kk